### DISCUSSION TITLE
Add system settings (system.yml) to version control?

### DISCUSSION THREAD

**Discussion by VincentBibauw** [pimcore/pimcore#13802] [Q&A / Support] [answered]
Hi

Do you add the system settings (/var/config/system.yml) to version control?
By default, this is added to the .gitignore, what's the reason for this?

This file does important setting for the project, and does not contain sensitive information, so I would add this to the VCS.

**Comment by blankse** [comment, marked as answer]
I think because the admin users can change this file in the admin interface via the system settings. So you have changes in this file in the livesystem. We removed it from our .gitignore and make no changes via admin interface. You can make this settings also in the config/config.yaml

**Reply by VincentBibauw** [reply]
Other changes in the admin interface also make changes in files in the livesystem, and those files are added to git by default. 
Anyway, we'll also remove this file from the .gitignore.
Thanks!
